kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -83,4 +86,34 @@ object TypelevelKernelPlugin extends AutoPlugin {
     }
   }
 
+  private[sbt] lazy val currentRelease: Def.Initialize[Option[String]] = Def.setting {
+    // some tricky logic here ...
+    // if the latest release is a pre-release (e.g., M or RC)
+    // and there are no stable releases it is bincompatible with,
+    // then for all effective purposes it is the current release
+
+    val release = previousReleases.value match {
+      case head :: tail if head.isPrerelease =>
+        tail
+          .filterNot(_.isPrerelease)
+          .find(head.copy(prerelease = None).mustBeBinCompatWith(_))
+          .orElse(Some(head))
+      case releases => releases.headOption
+    }
+
+    release.map(_.toString)
+  }
```

Comment:
Shoot, still not quite right 😂 I thought this was using `version` in here, but you are right that it's not. Sorry, I was confusing with the change I made in https://github.com/typelevel/sbt-typelevel/pull/282 which does use `version`.

So as it's currently defined, even if you've bumped your base version, if you haven't actually tagged anything with that new base version then it will still use whatever the latest tag is under the old base version. I guess this makes sense. But, now to explain it 🤔 **tl;dr** we do some blackbox heuristics to get you a sensible number, take it or leave it 😛